js-exercises/007_arrayAndConstant.js

```diff
@@ -0,0 +1,14 @@
+const readline = require("readline-sync");
+
+let arr = [];
+let result = [];
+arr = readline.question("Enter array decimal elements separated by comma:");
+
+let constant = readline.question("Enter constant value :");
+const addConstant = (arr, constant) => {
+  result = arr.split(",").map(el => {
+    return constant * parseInt(el);
```

Comment:
Why you are multiplying

Reply:
it's my mistake by giving name of PR.it is actually it is Multiply array element with a constant value.

Reply:
change the name of the function then